hw/bsp/samd11_xplained/board.mk

```diff
@@ -0,0 +1,53 @@
+CFLAGS += \
+  -ffunction-sections \
+  -fdata-sections \
+  -Wl,--gc-sections \
+  -mthumb \
+  -mabi=aapcs-linux \
+  -mcpu=cortex-m0plus \
+  -nostdlib -nostartfiles \
+  -D__SAMD11D14AM__ \
+  -DCONF_DFLL_OVERWRITE_CALIBRATION=0 \
+  -DOSC32K_OVERWRITE_CALIBRATION=0 \
+  -DCFG_TUSB_MCU=OPT_MCU_SAMD21 \
```

Comment:
you could add `-DCFG_EXAMPLE_MSC_READONLY` to allow using ROM instead of RAM for read-only mass storage device. It can still be enumerated and show up the readme though PC won't be able to change the disk and/or readme contents. Here is how other small chip define this.  
https://github.com/hathach/tinyusb/blob/master/hw/bsp/lpcxpresso1347/board.mk#L9

The macros is kind of required as well to pass CI build test.